index.html

```diff
@@ -11,51 +11,187 @@
+                <img src="./images/greenline.png" alt="greenline">
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
```

Comment:
- [ ] It would be great to apply the button interactions, as shown above, 👍 for improving the User Interface.